### DISCUSSION TITLE
No module named 'finrl_meta'

### DISCUSSION THREAD

**Discussion by Arishi-Orra** [AI4Finance-Foundation/FinRL#967] [Q&A]
I have successfully installed the FinRL library and when I am running the [Stock_NeurIPS2018.ipynb] (https://github.com/AI4Finance-Foundation/FinRL-Tutorials/blob/master/1-Introduction/Stock_NeurIPS2018_SB3.ipynb) file, it showing the following error:

1. "
from finrl_meta.preprocessor.yahoodownloader import YahooDownloader

ModuleNotFoundError: No module named 'finrl_meta'
"

The same error is also pooping when running the following command:
2."
from finrl.meta.preprocessor.preprocessors import FeatureEngineer, data_split

ModuleNotFoundError: No module named 'finrl.meta'
"

Also, one more error:
3."    from finrl import config_tickers

ImportError: cannot import name 'config_tickers' from 'finrl' 
"

Need help with what I am doing wrong here.

**Comment by Himanshu-Choudhary-1** [comment]
I am also facing the same issue...

**Comment by swtormy** [comment]
Perhaps you are installing the library with `pip install finrl`? Try use the unstable version of `pip install git+https://github.com/AI4Finance-Foundation/FinRL.git` as described in the [documentation](https://finrl.readthedocs.io/en/latest/start/installation.html)
